test/core/tsi/transport_security_test_lib.cc

```diff
@@ -303,6 +303,7 @@ void tsi_test_frame_protector_receive_message_from_peer(
 grpc_error_handle on_handshake_next_done(
     tsi_result result, void* user_data, const unsigned char* bytes_to_send,
     size_t bytes_to_send_size, tsi_handshaker_result* handshaker_result) {
+  LOG(ERROR) << "anasalazar";
```

Comment:
In tests, unlike the production code, you can leave log messages in place if you'd like.  However, please make the messages say something that will actually be useful, and don't log at level ERROR unless there's actually an error.